CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ following [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+## [0.5.0] - 2023-10-10
+
```

Comment:
anything to report? :)

Reply:
aha! just missed it for a few seconds

Reply:
I think mentioning general "dependency updates" is OK